docs/source/install/overview.rst

```diff
@@ -0,0 +1,62 @@
+Overview: Single-box Reference Deployment
+=========================================
+
+.. todo:: short intro paragraph: this is reference deployment for single box, follow installation
+    guide to get here.
+
+.. figure:: https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/pub?w=960&amp;h=720
+    :align: center
+
+    StackStorm single-box reference deployment.
+
+.. source https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/edit
+
+.. todo:: keep image live `on google.drive <https://docs.google.com/drawings/d/1X6u8BB9bnWkW8C81ERBvjIKRfo9mDos4XEKeDv6YiF0/edit>`_ while WIP but copy a final .png under _static
+
+1. st2* services
+----------------
+st2* services provide main stackstorm functionality. They are located at ``/opt/stackstorm/st2``, share a dedicated Python virtualenv, and configured by /etc/st2/st2.conf.
+
+    * st2sensorcontainer runs sensor from ``/opt/stackstorm/packs``.
```

Comment:
As I understand from new path hierarchy, it should be `/opt/stackstorm/st2/packs`